database.py

```diff
@@ -29,6 +36,15 @@ def get_region(self):
     def get_path(self):
         return self.__path
 
+    def get_pkmn_type(self):
+        return self.__pkmn_type
+
+    def get_pkmn_type_secondary(self):
+        return self.__pkmn_type_secondary
+
+    def get_dark_threshold(self):
```

Comment:
Can we please make these `@property`s?  Pokémon.type, Pokémon.type_secondary, and Pokémon.dark_threshold are so much easier to understand without the `get_` and the `()`.